File: SISMUEBLUM JAVA/Producto.java
public class Producto{
	private String nombre;
	private String color;
	private Proveedor marca;
	private String familia;
	private String codigo;
	private double precio;
	
	public Producto(String nombre, String color, Proveedor marca, String familia, String codigo, double precio){
		this.nombre = nombre;
		this.color = color;
		this.familia = familia;
		this.codigo = codigo;
		this.marca = marca;
		this.precio = precio;
	}
	
	public String getNombre () {return this.nombre;}
	public void setNombre (String valor) {this.nombre = valor;}
	public String getColor() {return this.color;}
	public void setColor (String valor) {this.color = valor;}
	public String getFamilia () {return this.familia;}
	public void setFamilia (String valor) {this.familia = valor;}
	public String getCodigo () {return this.codigo;}
	public void setCodigo (String valor) {this.codigo = valor;}
	public Proveedor getMarca () {return this.marca;}
	public void setMarca (Proveedor valor) {this.marca = valor;}
	public double getPrecio () {return this.precio;}
	public void setPrecio (double valor) {this.precio = valor;}	
}

File: SISMUEBLUM C#/Producto.cs
using System;
public class Producto{
	private string _nombre;
	private string _color;
	private Proveedor _marca;
	private string _familia;
	private string _codigo;
	private double _precio;
	
	public Producto(string nombre, string color, Proveedor marca, string familia, string codigo, double precio){
		this._nombre = nombre;
		this._color = color;
		this._familia = familia;
		this._codigo = codigo;
		this._marca = marca;
		this._precio = precio;
	}
	
	~Producto(){}
	
	public string nombre{get{return this._nombre;} set{this._nombre = value;}}
	public string color{get{return this._color;} set{this._color = value;}}
	public Proveedor marca{get{return this._marca;} set{this._marca = value;}}
	public string familia{get{return this._familia;} set{this._familia = value;}}
	public string codigo{get{return this._codigo;} set{this._codigo = value;}}
	public double precio{get{return this._precio;} set{this._precio = value;}}
}